stars/star_system.py

```diff
@@ -52,13 +56,22 @@ def create_system(self, race=None, num_planets=-1):
             segment = 100.0 / num_planets
             planet_args['ID'] = self.ID + ' ' + _roman[i]
             planet_args['distance'] = round(segment * i + randint(5, round(segment)))
-            if i != home:
+            planet_args['gravity'] = 'generate'
+            planet_args['temperature'] = 'generate'
+            if i != home - 1:
```

Comment:
I'm trying to understand the -1 one part

Reply:
0 based counting

Reply:
its because the sun is planet 0 and num_planets is planets besides the sun. If the sun is the home world then the i == home -1 will never get called because home==0 and range starts at 0. When home==-1 it doesn't have a race so isn't creating a home world.

Reply:
```suggestion
            if i + 1 != home:
```